apps/web/src/components/ScreeningDecisions/ScreeningDecisionDialogForm.tsx

```diff
@@ -0,0 +1,149 @@
+import * as React from "react";
+import { useIntl } from "react-intl";
+import { useFormContext } from "react-hook-form";
+
+import {
+  AssessmentDecision,
+  AssessmentResult,
+  AssessmentResultJustification,
+  Maybe,
+} from "@gc-digital-talent/graphql";
+import { CardOptionGroup, Checklist, TextArea } from "@gc-digital-talent/forms";
+import { errorMessages } from "@gc-digital-talent/i18n";
+
+import useLabels from "./useLabels";
+import { DialogType } from "./useDialogType";
+import useOptions from "./useOptions";
+
+const TEXT_AREA_ROWS = 3;
+const TEXT_AREA_MAX_WORDS = 200;
+
+export type FormValues = {
+  assessmentDecision: AssessmentResult["assessmentDecision"];
+  justifications: AssessmentResult["justifications"];
+  assessmentDecisionLevel: AssessmentResult["assessmentDecisionLevel"];
+  otherJustificationNotes: AssessmentResult["otherJustificationNotes"];
+  skillDecisionNotes: AssessmentResult["skillDecisionNotes"];
+  notesForThisAssessment: Maybe<string>; // TODO: Does this field need to be added to AssessmentResult model?
```

Comment:
This modal in the issue

Reply:
This is a different field then `skillDecisionNotes`, and it hasn't been added on the backend yet. Should I add it to the AC of this issue, or should we create a separate issue for this?

Reply:
Let's first see if Peter has any thoughts on the matter, if needed, yeah lets split off an issue to rather than scope creep. 
It'd need addition in a few places so best left to be its own thing.